pylearn2/linear/tests/test_cudnn.py

```diff
@@ -0,0 +1,151 @@
+"""
+Tests for the Cudnn code
+"""
+
+import theano
+from theano import tensor
+from theano.sandbox.cuda.dnn import dnn_available
+
+from pylearn2.linear.conv2d import Conv2D
+from pylearn2.linear.cudnn2d import Cudnn2D, make_random_conv2D
+from pylearn2.space import Conv2DSpace
+from pylearn2.utils import sharedX
+from pylearn2.testing.skip import skip_if_no_gpu
+
+import unittest
+from nose.plugins.skip import SkipTest
+
+import numpy as np
+
+
+class TestCudnn(unittest.TestCase):
+    """
+    Tests for the Cudnn code
+    """
```

Comment:
It's complaining about a missing "Parameters" section here, I'm not really sure why.
Maybe you can put a Parameters section that just says to refer to unittest.TestCase and check it fixes the error.